packages/errors/src/messages.ts

```diff
@@ -17,6 +18,7 @@ export const SolanaErrorMessages: Readonly<{
     // TypeScript will fail to build this project if add an error code without a message.
     [P in SolanaErrorCode]: string;
 }> = {
+    [SOLANA_ERROR__INVALID_KEYPAIR_BYTES]: 'KeyPair bytes must be of length 64',
```

Comment:
Slight preference for…
```suggestion
    [SOLANA_ERROR__INVALID_KEYPAIR_BYTES]: 'Key pair bytes must be of length 64',
```